CONTRIBUTING.md

```diff
@@ -1,27 +1,88 @@
+
+In order to contribute via GitHub you'll need to set up a free account and sign in. Here are some [instructions](https://help.github.com/articles/signing-up-for-a-new-github-account/) to help you get going. Remember that you can ask us any questions you need to along the way.
+
+## Writing in markdown
+
+GitHub has a helpful page on [getting started with writing and formatting on GitHub](https://help.github.com/articles/getting-started-with-writing-and-formatting-on-github).
+
+Most of the writing that you'll do will be in [Markdown][markdown]. You can think of Markdown as a few little symbols around your text that will allow GitHub to render the text with a little bit of formatting. For example you could write words as **bold**, or in *italics*, or as a [link][rick-roll] to another webpage.
 
 ## Where to start: wiki, code and templates
 
-Wiki: The main part of the BIDS starters kit is the wiki. It explains the first steps in creating your own BIDS compatible dataset. 
+#### [Wiki][bids-starterkit-wiki]
 
-Code: This repository is under development, and should contain the first steps of simple code (Matlab, Python R...) to read and write create BIDS-sidecar files. 
+We hope that the easiest place to find information about BIDS is the [**starter kit wiki**][bids-starterkit-wiki].
 
-Templates: This repository is under development, but we aim to have two templates for each BIDS sidecar file: one with a full example (all required/recommended/optional fields), and one with a minimal example (only required fields).
+You only need to be logged in to GitHub to edit the wiki. So, there's no need for a pull request if you just want to fix a typo or add a useful link!
+
+Here's a useful [introduction to GitHub wikis][intro-github-wiki]. Have a read through the pages that already exist in the wiki and please EDIT AWAY! :muscle::boom::sparkles:
+
+#### [Code][bids-starterkit-repo]
+
+This repository is under development, but we hope that it becomes a useful place for people to share and find small snippets of code that are useful for getting started with BIDS.
+
+To contribute to the codebase you'll need to submit a pull request. Check out our [guidelines](#making-a-change-with-a-pull-request) below.
+
+#### [Templates][bids-starterkit-repo]
+
+This repository is under development, but we aim to have two templates for each BIDS sidecar file: one with a full example (all required/recommended/optional fields), and one with a minimal example (only required fields).
+
+To contribute a template you'll need to submit a pull request. Check out our [guidelines](#making-a-change-with-a-pull-request) below.
 
 ## Where to start: labels
 
-The list of labels for current issues are the same as for the BIDS [here](https://github.com/INCF/BIDS/labels) and include:
+<!--
+  COMMENTED OUT BY KW because she's changed the sentence below!
+  ******* NEED TO CHECK W EMDP *******
```

Comment:
I must have written this a while back, when we didn't have many issues on the starter kit itself ! I definitely think your revision makes sense, now :smile:

Reply:
:+1: Great -thank you!